Module1.TypesAndClasses/Module1.TypesAndClasses.Tests/ShapesLinqTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Module1.TypesAndClasses.Interfaces;
+using Module1.TypesAndClasses.Shapes;
+using Module1.TypesAndClasses.Extensions;
+using Moq;
+using Xunit;
+using System.Linq;
+
+
+namespace Module1.TypesAndClasses.Tests
+{
+
+    public class ShapesLinqTests
+    {
+        private List<IShape> shapes = new List<IShape>
+            {
+                new Circle(2,Units.meters),
+                new EquilateralTriangle(5),
+                new Rectangle(4,4, Units.meters),
+                new Rectangle(40,50, Units.centimeters),
+                new Rectangle(600,400, Units.centimeters),
+                new Rectangle(400,200, Units.centimeters),
+                new Rectangle(4000,8000, Units.millimeters)
+            };
```

Comment:
in this case ok, because there are no mocks here, just real data.
If we had some mocks, then it should be inside constructor